### PR TITLE
Updated, improved and fixed Dutch language strings

### PR DESCRIPTION
**Pull request by ghost**
Not 100% complete yet. There are still some strings untranslated.

### CODE REVIEW

**Review by wxcafe — APPROVED** [review]

**Review by mjankowski — CHANGES_REQUESTED** [review]
There's a parsing issue in  here (see CI failure)

**Comment by jeroenpraat** [comment]
@mjankowski I don't understand what can be wrong. And I don't see anything helpful in CI.

**Comment by mjankowski** [comment]
Missing comma at line end here:

    "navigation_bar.blocks": "Geblokkeerde gebruikers"

**Comment by jeroenpraat** [comment]
@mjankowski Fixed.

**Review by Gargron — APPROVED** [review]

**Comment by jeroenpraat** [comment]
Again CI failed, but when I look into the log I see: Unused keys (17) | i18n-tasks v0.9.13' - That's not true, cause those keys where added a little while ago.

File: config/locales/nl.yml [hunk 1]
```diff
@@ -152,10 +154,61 @@ nl:
...
     disable: Uitschakelen
     enable: Inschakelen
-    instructions_html: "<strong>Scan deze QR-code in Google Authenticator of een soortgelijke app op je mobiele telefoon</strong>. Van nu af aan creëert deze app tokens die je bij aanmelden moet invoeren."
+    instructions_html: "<strong>Scan deze QR-code in Google Authenticator of een soortgelijke app op jouw mobiele telefoon</strong>. Van nu af aan genereert deze app aanmeldcodes die je bij het aanmelden moet invoeren."
   users:
-    invalid_email: Het e-mailadres is ongeldig
-    invalid_otp_token: Ongeldige twee-factorcode
+    invalid_email: E-mailadres is ongeldig
+    invalid_otp_token: Ongeldige tweestaps-aanmeldcode
+  will_paginate:
```
Comment (mjankowski) [inline config/locales/nl.yml]: You can remove this.

**Comment by mjankowski** [comment]
It's confusing, but that error is accurate. Some things used to be under `reports` and are now nested under `admin.reports`

File: config/locales/nl.yml (hunk 1 above)
Comment (jeroenpraat) [inline config/locales/nl.yml]: What exactly? Line 162-164?

Comment (jeroenpraat) [inline config/locales/nl.yml]: Or 161?

Comment (mjankowski) [inline config/locales/nl.yml]: 164-165

**Comment by jeroenpraat** [comment]
Yes, it has passed!

**Review by Gargron — APPROVED** [review]

**Pull request merged**